src/infrastructure/content/IContentNotificationHandler.java

```diff
@@ -1,4 +1,4 @@
-package content;
+package infrastructure.content;
 
 public interface IContentNotificationHandler {
 	// Meta field is username of a new user
```

Comment:
Refer badal's interface for writing javadoc for functions. Remove the space between the function name and () brackets

Reply:
All the parameters are JSON strings